src/timeseriesinsights/azext_timeseriesinsights/_params.py

```diff
@@ -0,0 +1,89 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+# pylint: disable=line-too-long
+# pylint: disable=too-many-lines
+# pylint: disable=too-many-statements
+
+from azure.cli.core.commands.parameters import (
+    name_type,
+    tags_type,
+    get_enum_type,
+    resource_group_name_type,
+    get_location_type
+)
+from azure.cli.core.commands.validators import get_default_location_from_resource_group
+
+from knack.arguments import CLIArgumentType
+from ._validators import iso_8601_timespace
+
+environment_name_type = CLIArgumentType(help='The name of the Time Series Insights environment associated with the specified resource group.')
+
+
+def load_arguments(self, _):
+
+    with self.argument_context('timeseriesinsights operation list') as c:
+        pass
+
+    with self.argument_context('timeseriesinsights') as c:
+        c.argument('resource_group_name', resource_group_name_type)
```

Comment:
We can remove all `resource_group_name` definition here if there is no special logic for it. And I'm curious about why some are defined as `resource_group_name` and some some called `resource_group` ?

Reply:
```suggestion
```